module4/homework/appendNewRecipe/AppendNewRecipe.cpp

```diff
@@ -0,0 +1,55 @@
+#include "AppendNewRecipe.hpp"
+
+#include <algorithm>
+#include <fstream>
+#include <iterator>
+#include <map>
+
+std::map<char, std::string> units{
+    {'g', "gram"},
+    {'s', "szklanka(i)"},
+    {'m', "mililitrow"}};
+
+bool AppendNewRecipe(VecStr steps, const ListStr& ingredients, const DeqSizetChar& amount) {
+    auto newRecipe = (FormatRecipit(steps, ingredients, amount)).str();
+
+    std::fstream file("recipes.txt", file.app);
```

Comment:
I suggest you to make files paths as constexpr char[].

Reply:
+1